test_scripts/RC/OnRCStatus/002_notification_by_rc_functionality_disallowed_on_hmi.lua

```diff
@@ -10,7 +10,7 @@
 -- 1) RC functionality is disallowed on HMI
 -- 2) RC application is registered
 -- SDL must:
--- 1) Not send OnRCStatus notification to registered mobile application and the HMI
+-- 1) Send OnRCStatus(allowed = false, freeModules = {}, allocatedModules = {}) notification to registered mobile application and the HMI
```

Comment:
why send to HMI?
Send OnRCStatus(allowed = false, freeModules = {}, allocatedModules = {}) notification to registered mobile `RC` applications